packages/apollo-shared/src/ChangeManager/ChangeManager.ts

```diff
@@ -86,9 +94,96 @@ export class ChangeManager {
     }
   }
 
+  async submitToClientOnly(change: Change) {
```

Comment:
Instead of making a new `submitToClientOnly`, this should use the existing `submit(change, {submitToBackend: false})`

Reply:
Done